pipelines/h6c/rtp/v2/stage_1_task_scripts/do_ALL_SKY_IMAGE_MOVIE.sh

```diff
@@ -0,0 +1,30 @@
+set -e
+
+# This script generates an HTML version of a notebook summarizing the output of auto_metrics, ant_metrics, and redcal chisq
+
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consistent with the config.
+# 1 - (raw) filename
+# 2 - nb_template_dir: where to look for the notebook template
+# 3 - nb_output_repo: repository for saving evaluated notebooks
+# 4 - git_push: boolean whether to push the results created in the nb_output_repo
+# 5 - good_statuses: string list of comma-separated (no spaces) antenna statuses considered "good"
```

Comment:
Out of date

Reply:
copy-pasta from the example out of H6C_v2